docs/openapi/api-external-b2b-webhook-v1.yaml

```diff
@@ -284,13 +405,13 @@ paths:
           content:
             application/problem+json:
               schema:
-                $ref: 'remote-refs.yaml#/components/schemas/Problem'
```

Comment:
Queste modifiche a cosa sono dovute?

Reply:
il ref: remote-refs.yaml non contiene la definizione dell'Entity "Problem", che è invece presente in schemas-pn-errors-v1.yaml. 
Provveduto ad inserire puntamento corretto del ref